Tests/MapboxMapsTests/Gestures/GestureHandlers/PinchGestureHandlerTests.swift

```diff
@@ -143,6 +143,69 @@ final class PinchGestureHandlerTests: XCTestCase {
         XCTAssertNil(mapboxMap.setCameraStub.parameters[2].bearing)
     }
 
+    func testPinchChangedWithZoomEnabledFalse() {
+        pinchGestureHandler.zoomEnabled = false
+
+        mapboxMap.cameraState = .random()
+
+        // these touches are consistent with the initial pinch midpoint
+        // and yield an angle of 45° for the initial state
```

Comment:
This comment was carried over from another test, but doesn't apply here since we're not configuring the touch locations

Reply:
This isn't fully addressed.